### ISSUE TITLE
Manage scu_dma_cpu_level_sanitize errors

### ISSUE THREAD

**Issue by shazz** [ijacquez/libyaul#6]
Currently if something bad is detected in the sanitizer, it does nothing that can tell the caller something went wrong :
```
if ((scu_dma_cpu_level_sanitize(cfg, mode)) < 0)
                return;
```
For example if the indirect table is empty or not well aligned or whatever, it just exit. So would be better to add assert or a return code in 
```
void
scu_dma_cpu_level_set(enum dma_level lvl, enum dma_mode mode, struct dma_level_cfg *cfg)
```

**Comment by ijacquez** [comment]
Good idea. Though, it would make sense if `scu_dma_cpu_level_sanitize()` is only built when building debug. This way, there is no sanity check overhead.
